test/integration/model.test.js

```diff
@@ -2587,6 +2592,12 @@ describe(Support.getTestDialectTeaser('Model'), () => {
             break;
           }
 
+          case 'yugabytedb': {
```

Comment:
Add this `case 'yugabytedb':` before `case 'postgres':` to avoid duplicating their contents